### DISCUSSION TITLE
Does ory-hydra provide PAR OAuth 2.0 (RFC 9126) implementation?

### DISCUSSION THREAD

**Discussion by epicaqua** [ory/hydra#3071] [Q&A] [answered]
Hello there, 
would ory-hydra also provide PAR(Pushed Authorization Request) OAuth 2.0 (https://datatracker.ietf.org/doc/rfc9126/) rfc implementation soon? is it already there somewhere?

**Comment by vinckr** [comment]
Transferring this discussion to ory/hydra.

I don't think Pushed Authorization Requests are implemented or planned to be implemented at the moment. 
Feel free to open a feature request for this, but it would probably be good to understand what you want to achieve and why Pushed Authorization Requests are a good solution / a feature to be added to Ory Hydra, this would help the maintainers and community to understand this request better,

thanks!
 
ps: Ory Hydra will be available in Ory Cloud soon.

**Comment by aeneasr** [comment, marked as answer]
To extend @vinckr answer - not yet but there’s a PR open for it: https://github.com/ory/fosite/pull/660

**Reply by justayar** [reply]
I see that this feature was merged to master. Is the PAR feature planned to integrate into the ory/hydra project?

**Reply by mchristophe** [reply]
@aeneasr : same question as @justayar  :)
